syntax = "proto3";

enum RequestType {
	BLOB = 0;
}

message Request {
	RequestType request_type = 1;
	bytes request_message = 2;
}

message BlobMessage {
    double timestamp = 1;
    bytes blob = 2;
}
